specifyweb/permissions/migrations/0007_add_stats_edit_permission.py

```diff
@@ -4,16 +4,16 @@
 
 
 def add_stats_edit_permission(apps, schema_editor):
-    for collection in Collection.objects.all():
+    for collection_id in Collection.objects.values_list('id', flat=True):
         try:
-            all_full_access_roles = Role.objects.filter(collection=collection, name="Full Access - Legacy")
+            all_full_access_roles = Role.objects.filter(collection__id=collection_id, name="Full Access - Legacy")
```

Comment:
minor, don't have double underscore. doesn't matter, bc I think it optimizes it out, but to avoid confusion downstream